pkg/virtctl/vm/vm.go

```diff
@@ -389,9 +433,88 @@ func getDryRunOption(dryRun bool) []string {
 	return nil
 }
 
-func (o *Command) Run(args []string) error {
+func expandArgs() cobra.PositionalArgs {
+	return func(cmd *cobra.Command, args []string) error {
+		if filePath == "" && vmName == "" {
```

Comment:
Is it possible to mark either one of the `--file` or `--vm` flags required, so you can omit this check?

Reply:
(while keeping them mutually exclusive)

Reply:
How would that work? If `--file` is required you couldn't provide only `--vm`?

Reply:
We discussed this one and decided not to change it.